test/smoke/go/kogitoapp.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/controller/kogitoapp/resource"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/util"
+)
+
+const (
+	KOGITO_EXAMPLES_URI = "https://github.com/kiegroup/kogito-examples"
+)
+
+func DeployNonNativeQuarkusExample(namespace, appName, contextDir string) error{
+	fmt.Println("Deploy non native quarkus example " + contextDir + " with name " + appName)
```

Comment:
can't we import the logging here?

Reply:
I used the logger package in the test files